bitbots_blackboard/src/bitbots_blackboard/capsules/team_data_capsule.py

```diff
@@ -12,53 +12,62 @@ class TeamDataCapsule:
     def __init__(self):
         self.bot_id = rospy.get_param("bot_id", 1)
         self.strategy_sender = None  # type: rospy.Publisher
-        self.team_data = TeamData()
+        # indexed with one to match robot ids
+        self.team_data = [None, TeamData(), TeamData(), TeamData(), TeamData(), TeamData(), TeamData()]
```

Comment:
You could use a Default Dict here

Reply:
I dont think a default dict is necessary, but a dict is probably cleaner than this hacky 1 indexed array. If a wrong robot_id is used, I think it is okay to throw an error